### ISSUE TITLE
loss goes to zero with hinge loss

### ISSUE THREAD

**Issue by raingo** [BVLC/caffe#1605]
Hi,

I am new to Caffe. When I try to train on ImageNet 2012 using hinge loss, the loss goes to zero directly. I guess there are some numerical issues. The configuration I am using is exactly the same as bvlc_reference_imagenet, except the loss is HINGE_LOSS. Could anyone share the configuration to do this? Ideally, some insights will be very helpful.

Thanks,
Raingo

**Comment by shelhamer** [comment]
Please ask modeling questions on the [caffe-users](https://groups.google.com/forum/#!forum/caffe-users) mailing list. Thanks!

**Comment by sronen71** [comment]
I had the same problem, but I found the reducing the learning rate resolved it.
In my case it was necessary to reduce the learning rate by 1/4 compared to softmeax loss to get it work.
